core/src/toga/app.py

```diff
@@ -349,6 +348,29 @@ def __init__(
     def _create_impl(self):
         return self.factory.App(interface=self)
 
+    @property
+    def paths(self):
+        """
+        Paths for known safe system locations. Using paths makes sure you have a consistent and known location to store
```

Comment:
Line length is still an issue - it should be wrapped at ~80 chars. 

There's also no mention of Pathlib as a return type, as previously requested.

Reply:
Can I instead suggest that we wrap at 88 columns, the same as Black? That makes it much easier to set one line length in your editor and use it everywhere.